src/projetos/HomeProjetosListPage.tsx

```
@@ -0,0 +1,99 @@
+            <button
+              type="button"
+              className="!bg-transparent !text-gray-500 !p-0 !m-0 !border-none !shadow-none">
+              Projetos Finalizados
+            </button>
+
+          </TabsList>
+
+          <TabsContent value={tab} className="mt-4">
+            <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6">
+              {projetos.length > 0 ? (
+                projetos.map((projeto) => <ProjetoCard key={projeto.id} projeto={projeto} />)
```

ESLint

[error] Replace `<ProjetoCard·key={projeto.id}·projeto={projeto}·/>` with `(⏎··················<ProjetoCard·key={projeto.id}·projeto={projeto}·/>⏎················)`

(prettier/prettier)